tests/feature_tests/pivoting/Makefile

```diff
@@ -13,6 +13,9 @@ fetch_manifests:
 pivoting:
 	./pivot.sh
 
+fetch_target_logs:
+	./../../scripts/fetch_target_logs.sh
```

Comment:
It would be more readable and maintainable if path would not be relative, but from the repository root.

Reply:
Agreed to be fixed in another PR, following the convention in the script.